docs/resources/gaussdb_redis_instance.md

```diff
@@ -0,0 +1,179 @@
+---
+subcategory: "GaussDB"
+---
+
+# huaweicloud\_gaussdb\_redis\_instance
+
+GaussDB for Redis instance management within HuaweiCoud.
+
+## Example Usage
+
+### create a gaussdb for redis instance with tags
+
+```hcl
+resource "huaweicloud_gaussdb_redis_instance" "instance_1" {
+  name              = "gaussdb_redis_instance_1"
+  password          = var.password
+  flavor            = "geminidb.redis.xlarge.4"
+  volume_size       = 100
+  vpc_id            = var.vpc_id
+  subnet_id         = var.subnet_id
+  security_group_id = var.secgroup_id
+  availability_zone = var.availability_zone
+
+  tags = {
+    foo = "bar"
+    key = "value"
+  }
+}
+```
+
+### create a gaussdb redis instance with backup strategy
+
+```hcl
+resource "huaweicloud_gaussdb_redis_instance" "instance_1" {
+  name              = "gaussdb_redis_instance_1"
+  password          = var.password
+  flavor            = "geminidb.redis.xlarge.4"
+  volume_size       = 100
+  vpc_id            = var.vpc_id
+  subnet_id         = var.subnet_id
+  security_group_id = var.secgroup_id
+  availability_zone = var.availability_zone
+
+  backup_strategy {
+    start_time = "03:00-04:00"
+    keep_days  = 14
+  }
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the Redis instance resource.
+  see [Endpoints](https://developer.huaweicloud.com/endpoint) for more detail.
+  If omitted, the provider-level region will be used. Changing this creates a new Redis instance resource.
+
+* `availability_zone` - (Required, String, ForceNew) Specifies the AZ name.
+  see [Endpoints](https://developer.huaweicloud.com/endpoint) for more detail.
+  Changing this parameter will create a new resource.
+
+* `name` - (Required, String) Specifies the instance name, which can be the same
+  as an existing instance name. The value must be 4 to 64 characters(Chinese characters are allowed) in
```

Comment:
This description (Chinese characters are allowed) is unnecessary because letter contains ENGLISH character and CHINESE character. Suggest add `Chinese characters must be in UTF-8 or Unicode format.` at the end of the name descriptions.

Reply:
updated now